sympy/polys/orthopolys.py

```diff
@@ -70,8 +69,10 @@ def dup_gegenbauer(n, a, K):
     return seq[n]
 
 
-def gegenbauer_poly(n, a, x=None, **args):
-    """Generates Gegenbauer polynomial of degree `n` in `x`. """
+def gegenbauer_poly(n, a, x=None, polys=False):
+    """Generates Gegenbauer polynomial of degree `n` in `x`.
+    Returns a Poly object when polys=True, otherwise
```

Comment:
similarly here `(default)` in brackets after "otherwise"

Also the double backticks.